features/unit/v2algodclient_paths.feature

```diff
@@ -3,48 +3,55 @@ Feature: Algod REST API v2
   # SendRaw, Status, Supply, SuggestedParams, and PendingTransactionsInformation omitted - the path never mutates, they're constant
 
   Scenario Template: Shutdown
+    Given mock server recording request paths
```

Comment:
You can use a background to define this on all scenarios: https://relishapp.com/cucumber/cucumber/docs/gherkin/background

```
  Background:
    Given mock server recording request paths
```